### PR TITLE
chore(ci): Stop Debian package failing if vector user already exists

### LINKED ISSUE #5919: Reinstalling deb fails because vector user already exists
This looks to be due to #5666 as `adduser` is content to continue if the user already exists.

 cc/ @jamtur01

### PR DESCRIPTION
**Pull request by jamtur01**
Closes #5919 

Signed-off-by: James Turnbull <james@lovedthanlost.net>

### CODE REVIEW

File: distribution/debian/scripts/preinst [hunk 1]
```diff
@@ -2,7 +2,7 @@
 set -e
 
 # Add vector:vector user & group
-useradd -M --system --user-group vector
+useradd -M --system --user-group vector || true
```
Comment (jszwedko) [inline distribution/debian/scripts/preinst:5]: I'm a bit concerned that this command could fail for other reasons that we might actually want to abort from. Maybe we could wrap it in a conditional check for the `vector` user's existence?

Just for my edification is there a reason we switched from `adduser`?

Comment (jamtur01) [reply distribution/debian/scripts/preinst:5]: Sure.

`adduser` isn't present on all the platforms where people use the *.deb packages - useradd is lower-level and more widely supported.

**Comment by binarylogic** [comment]
Could this also be tested as part of the verification steps? I assume installing twice would do it?

**Review by jszwedko — APPROVED** [review]
:+1: the implementation looks good to me. I'd support testing installing the package twice as @binarylogic suggested.

**Pull request merged**